### PR TITLE
Add "workload" to glossary

### PR DESCRIPTION
**Pull request by sftim**
Associated with #5993.

Despite being widely used in a cloud computing context, “workload” in an IT sense isn't particularly well defined. What we have here is my interpretation of a nebulous topic.
Given that, if you're viewing or reviewing this PR, I'd welcome constructive feedback on both the Markdown-level changes and the meaning of  this concept to typical Kubernetes users.

### CODE REVIEW

File: content/en/docs/reference/glossary/workload.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+date: 2019-02-12
+full_link: /docs/concepts/workloads/
+short_description: >
+  A set of applications for processing information to serve a user need.
+
+aka:
+tags:
+- workload
+---
+A workload consists of a system of services or applications that can run to fulfil a
```
Comment (zparnold) [inline content/en/docs/reference/glossary/workload.md]:
```suggestion
A workload consists of a system of services or applications that can run to fulfill a
```

File: content/en/docs/reference/glossary/workload.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Workload
+id: workload
+date: 2019-02-12
+full_link: /docs/concepts/workloads/
+short_description: >
+  A set of applications for processing information to serve a user need.
```
Comment (zparnold) [inline content/en/docs/reference/glossary/workload.md]:
```suggestion
  A set of applications for processing information to serve a purpose that is valuable to a single user or group of users.
```

**Comment by zparnold** [comment]
@sftim Made some suggestions PTAL

**Comment by sftim** [comment]
How's this looking now?

**Comment by zparnold** [comment]
@sftim Great! Thanks!
/lgtm
/approve

**Pull request merged**